import "server/backend/proto/index_search_common_names.proto";

message RequestParams {
  optional int32 product_id = 1;
  optional int32 breed_id = 2;
  optional int32 supply_province_id = 3;
  optional int32 supply_city_id = 4;
  optional int32 supply_county_id = 5;
  optional int32 page_size = 6[default=20];
  optional int32 page_no = 7[default=0];

  optional IndexSearchCommonNames names = 8;
}
